Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Drawing.Utilities/Controls/Components/ColourHexadecimalTextBox.cs

```diff
@@ -21,20 +19,46 @@ public ColourHexadecimalTextBox()
             Text = string.Empty;
         }
 
-        //private void ColourHexadecimalTextBox_TextChanged(object sender, EventArgs e)
-        //{
-        //    ColorConverter converter = new ColorConverter();
+        private bool IsHexadecimal(string input)
+        {
+            Regex hex = new Regex("^[a-fA-F0-9]+$");
```

Comment:
Set this to Precompile, and also to be a field rather than computing it everytime a colour needs to be converted.

Reply:
I think I may have another way of doing it, but I need to play with it. The problem I have at the moment is allowing it to only accept 0 -9 & A - F.

Reply:
Doesn't a int converter have a built in (.net to crt redirect) string with hex format already ?

Reply:
Resolved